### PR TITLE
Hide subject when null

### PR DESCRIPTION
**Pull request by madmaximux**

### CODE REVIEW

**Comment by alanhg** [comment]
Thanks, I'll deal with it as fast.

**Comment by madmaximux** [comment]
I'm sure you can come up with something cleaner :p

File: 2fa-read-code/src/index.js [hunk 1]
```diff
@@ -19,20 +19,40 @@ const lookBackMinutes = process.env.look_back_minutes;
...
-            }
-          })
-        );
+        subject = readSubjectFromMessage(msg);
+        if (!subject) {
+          // `subject` is falsey, which includes `undefined` and `null`
+          // (and `""`, and `0`, and `NaN`, and [of course] `false`)
+          // https://stackoverflow.com/a/5101991
+          res.push(
+            utils.buildItem({
```
Comment (alanhg) [inline 2fa-read-code/src/index.js:28]: Too much duplicate code, I gave up on merging. I will fix it directly based on this issue. thanks again.

**Review by alanhg — CHANGES_REQUESTED** [review]

**Comment by madmaximux** [comment]
Much cleaner looking. Thanks!

(I don't have much JavaScript experience so didn't know about ternary operators.)

**Pull request closed without merging**